src/components/standard/article.tsx

```diff
@@ -103,10 +104,13 @@ const Standard = ({ item, children }: Props): JSX.Element => {
                 {children}
             </Body>
             {epicContainer}
-            <footer css={articleWidthStyles}>
+            <section id="tags" css={articleWidthStyles}>
                 <Tags tags={item.tags} format={item}/>
-            </footer>
+            </section>
         </article>
+        <footer css={[articleWidthStyles, footerWidthStyles]}>
+            <RelatedContent css={articleWidthStyles} content={item.relatedContent}/>
```

Comment:
I'm not sure this `css` prop is obeyed for non-built-in components 🤔. I can't see the styles being applied in the DOM.

```suggestion
            <RelatedContent content={item.relatedContent}/>
```